src/main/kotlin/com/mutualmobile/praxisspringboot/services/orgs/impl/UserProjectServiceImpl.kt

```diff
@@ -15,7 +18,13 @@ class UserProjectServiceImpl : UserProjectService {
     @Autowired
     lateinit var userProjectRepository: UserProjectRepository
 
-    override fun assignProjectToUser(projectId: String, userId: String): ResponseEntity<ApiResponse<HarvestUserProject>> {
+    @Autowired
+    lateinit var userWorkRepository: UserWorkRepository
+
+    override fun assignProjectToUser(
+        projectId: String,
+        userId: String
+    ): ResponseEntity<ApiResponse<HarvestUserProject>> {
```

Comment:
@shubhamsinghmutualmobile we need just one api which links multiple projects to multiple users in one request.